### PR TITLE
[RM Ch3] Infrastructure and app level acceleration

### LINKED ISSUE #2051: [RM Ch03/04] HW Acceleration abstraction
Need to define an abstract model of acceleration and map to some concrete acceleration technologies

### PR DESCRIPTION
**Pull request by petorre**
Fixes #2051 .

### CODE REVIEW

File: doc/ref_model/chapters/chapter03.md [hunk 1]
```diff
@@ -285,7 +285,29 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
...
 <a name="3.8.2"></a>
-### 3.8.2 Levels of accelerator consumption
+### 3.8.2 Infrastructure and Application Level Acceleration
+
+Figure 3-8 gives examples for Hardware Accelerators in [Sample reference model realization](#3.7) diagram.
+
+<p align="center"><img src="../figures/ch03-hardware-acceleration-in-rm-realization-diagram.png" alt="Hardware Acceleration in RM Realization Diagram" Title="Hardware Acceleration in RM Realization Diagram" width="65%"/></p>
+<p align="center"><b>Figure 3-6:</b> Hardware Acceleration in RM Realization Diagram</p>
+
+Hardware Accelerators are part of the Hardware Infrastructure Layer. Those that need to be activated/programmed will expose management interfaces and have Accelerator Management software reaching them in-band (from host OS) or out of band (OOB, over some network to the adapter without going through host OS). For more flexibility in management, such Accelerator Management can be carried over appropriate service with authentication mechanism before being exposed to Cloud Infrastructure operator and/or Application tenant.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter03.md:295]: "software reaching them in-band" ---> "software managing them in-band ..."

File: doc/ref_model/chapters/chapter03.md [hunk 2]
```diff
@@ -285,7 +285,29 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
 ### 3.8.1 Types of accelerators
 
 <a name="3.8.2"></a>
-### 3.8.2 Levels of accelerator consumption
+### 3.8.2 Infrastructure and Application Level Acceleration
+
+Figure 3-8 gives examples for Hardware Accelerators in [Sample reference model realization](#3.7) diagram.
+
+<p align="center"><img src="../figures/ch03-hardware-acceleration-in-rm-realization-diagram.png" alt="Hardware Acceleration in RM Realization Diagram" Title="Hardware Acceleration in RM Realization Diagram" width="65%"/></p>
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter03.md:292]: Please add the figure in .png and in editable version.

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter03.md (hunk 1 above)
Comment (petorre) [reply doc/ref_model/chapters/chapter03.md]: > 
> 
> "software reaching them in-band" ---> "software managing them in-band ..."

Changed. Thanks

File: doc/ref_model/chapters/chapter03.md (hunk 2 above)
Comment (petorre) [reply doc/ref_model/chapters/chapter03.md:292]: > Please add the figure in .png and in editable version.

@CsatariGergely : It is in #2162 . (Sorry, then I didn't know how in web UI to add it together with text, now I know.)

**Review by ulikleber — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Comment by walterkozlowski** [comment]
@TFredberg , @pgoyal01 - please review and approve if you can

**Review by pgoyal01 — APPROVED** [review]

**Comment by petorre** [comment]
Resolved conflict.

**Pull request merged**